### DISCUSSION TITLE
Bot stuck on "𝙔𝙤𝙪𝙧 𝙧𝙚𝙦𝙪𝙚𝙨𝙩 𝙞𝙨 𝙤𝙣 𝙥𝙧𝙤𝙘𝙚𝙨𝙨"

### DISCUSSION THREAD

**Discussion by lanairz** [shivaya-dav/DogeRat#61] [Q&A]
When I send a command (like main camera photo) to my phone, the bot sends nothing except: "°• 𝙔𝙤𝙪𝙧 𝙧𝙚𝙦𝙪𝙚𝙨𝙩 𝙞𝙨 𝙤𝙣 𝙥𝙧𝙤𝙘𝙚𝙨𝙨 • ʏᴏᴜ ᴡɪʟʟ ʀᴇᴄᴇɪᴠᴇ ᴀ ʀᴇꜱᴘᴏɴꜱᴇ ɪɴ ᴛʜᴇ ɴᴇxᴛ ꜰᴇᴡ ᴍᴏᴍᴇɴᴛꜱ"

**Comment by Billionzgaming1** [comment]
Same even get file not working

**Comment by Chaitu1710** [comment]
what is the solution ??
\
